Gemfile.lock

```diff
@@ -132,18 +132,6 @@ PATH
     veteran_confirmation (0.0.1)
     vye (0.1.0)
 
-GEM
```

Comment:
I don't think you meant to commit this?

Reply:
Uh, weird, I explicitly removed it. 

Removed and repushed regain?!